Packaging/amiga/prep.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+#set compiler params
+export TARGET='m68k-amigaos'
+export SYSROOT=/opt/$TARGET
+export M68K_CPU="-m68040 -mhard-float"
```

Comment:
buildroot has 68040 support btw:
https://github.com/buildroot/buildroot/blob/caff6dc4816c5279226c018ba10d5db71adb26eb/arch/Config.in.m68k#L23-L26